gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/LoadConfiguration.java

```diff
@@ -0,0 +1,381 @@
+     * By default no bad record is ignored.
+     */
+    public Builder maxBadRecords(Integer maxBadRecords) {
+      this.maxBadRecords = maxBadRecords;
+      return this;
+    }
+
+    /**
+     * Sets the schema for the destination table. The schema can be omitted if the destination table
+     * already exists, or if you're loading data from Google Cloud Datastore.
+     */
+    public Builder schema(Schema schema) {
+      this.schema = schema;
+      return this;
+    }
+
+    /**
+     * Sets whether BigQuery should allow extra values that are not represented in the table schema.
+     * If {@code true}, the extra values are ignored. If {@code true}, records with extra columns
+     * are treated as bad records, and if there are too many bad records, an invalid error is
+     * returned in the job result. By default unknown values are not allowed.
+     */
+    public Builder ignoreUnknownValues(Boolean ignoreUnknownValues) {
+      this.ignoreUnknownValues = ignoreUnknownValues;
+      return this;
+    }
+
+    /**
+     * Sets which entity properties to load into BigQuery from a Cloud Datastore backup. This field
+     * is only used if the source format is set to {@code DATASTORE_BACKUP}. Property names are case
+     * sensitive and must be top-level properties. If no properties are specified, BigQuery loads
+     * all properties. If any named property isn't found in the Cloud Datastore backup, an invalid
+     * error is returned in the job result.
+     */
+    public Builder projectionFields(List<String> projectionFields) {
+      this.projectionFields =
+          projectionFields != null ? ImmutableList.copyOf(projectionFields) : null;
+      return this;
+    }
+
+    public LoadConfiguration build() {
+      return new LoadConfiguration(this);
+    }
+  }
+
+  private LoadConfiguration(Builder builder) {
+    this.destinationTable = checkNotNull(builder.destinationTable);
+    this.createDisposition = builder.createDisposition;
+    this.writeDisposition = builder.writeDisposition;
+    this.formatOptions = builder.formatOptions;
+    this.maxBadRecords = builder.maxBadRecords;
+    this.schema = builder.schema;
+    this.ignoreUnknownValues = builder.ignoreUnknownValues;
+    this.projectionFields = builder.projectionFields;
+  }
+
+  /**
+   * Returns the destination table to load the data into.
+   */
+  public TableId destinationTable() {
+    return destinationTable;
+  }
+
+  /**
+   * Returns whether the job is allowed to create new tables.
+   *
+   * @see <a href="https://cloud.google.com/bigquery/docs/reference/v2/jobs#configuration.load">
+   *     Jobs: Load Configuration</a>
+   */
+  public CreateDisposition createDisposition() {
+    return this.createDisposition;
+  }
+
+  /**
+   * Returns the action that should occur if the destination table already exists.
+   *
+   * @see <a href="https://cloud.google.com/bigquery/docs/reference/v2/jobs#configuration.load">
```

Comment:
Nit: I think it would slightly clearer to link to the write disposition entry directly (#configuration.load.writeDisposition).  Same for `createDisposition()` and both methods' equivalents in the builder.